R/render.R

```diff
@@ -387,6 +387,12 @@ render <- function(input,
   }
   output_dir <- dirname(output_file)
 
+  # Stop the render process early if the output directory does not exist
+  if (!dir_exists(output_dir)) {
+    stop("The directory `", output_dir, "`) does not not exist.",
```

Comment:
nit: usually I used quotes, either double or single, instead of backticks in messages, since they feel more natural in text for humans. Backticks feel like Markdown-ish.

Reply:
Could I do a simple replacement of backticks to single quotes in a push to master?

Reply:
Yes, please go ahead. Thanks!